### PR TITLE
add detailed example to README.md

### LINKED ISSUE #35: Need to document common event sub and scripting patterns related to this gateway

### PR DESCRIPTION
**Pull request by krancour**
Fixes #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,10 +424,258 @@ event bus.
...
+
+In any case where the gateway forwards a `check_suite:rerequested` event to
+Brigade to be enqueued for any subscribed project, the gateway will also monitor
+the status of any and all jobs associated with each event and utilize the checks
+API to return test results to GitHub. In this way, the result of every job
+becomes the result of a single check (having the same name as the job) in the
+corresponding check suite. (This is how job results end up visible in the GitHub
+web UI.)
+
+In the event that any individual check in the suit fails, it can be re-requested
```
Comment (vdice) [inline README.md:485]:
```suggestion
In the event that any individual check in the suite fails, it can be re-requested
```

**Review by vdice — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (krancour) [reply README.md:485]: Nice catch. I am aware of other spelling mistakes as well, so I definitely need to take another pass through this.

**Comment by krancour** [comment]
I think the pattern that's documented here can be simplified further (as in #60), so I am going to defer these doc changes to v0.6.0 after the simpler pattern has proven itself with the v0.5.0 release.

In the meantime, I'll switch this to a draft.

**Comment by krancour** [comment]
Closing this in favor of #63

**Pull request closed without merging**